### ISSUE TITLE
Provide audio output

### ISSUE THREAD

**Issue by gerv** [mozilla/lightbeam#15]
If the aim of this is to freak people out about being tracked, how about a version with audio?

First, I thought: why don't I have this open in the corner while I'm browsing, so I can see the changes? Then I thought: it would be much better if I could _hear_ myself being tracked.

Imagine this: a low "bloop" when a new tracking website appears. Each time a link appears to an existing tracking website, you get a "bloop" one note up the scale from the last one. I.e. the pitch depends on how many links are already there.

In other words, after a bit of browsing, new links to Doubleclick would be high-pitched "beeps". As the pitch rises, so would the sense of someone keeping a very close eye on your browsing. As the user surfs around, they click a link and hear "boop" "beep" "beeeep" and think "Goodness, I click one link and three companies are tracking me!"

I guarantee this sort of thing will drive adoption of anti-tracking software. Try it! (Use<audio>, of course.)

Gerv
